src/Build/BackEnd/BuildManager/BuildParameters.cs

```diff
@@ -768,6 +773,15 @@ public string OutputResultsCacheFile
         /// </summary>
         public bool DiscardBuildResults { get; set; } = false;
 
+        /// <summary>
+        /// Gets or sets a value indicating whether the build process should run as low privilege.
```

Comment:
Ditto.

Reply:
Thanks for spotting this, not sure what happened there..